src/resources/views/fields/latlng.blade.php

```diff
@@ -0,0 +1,204 @@
+                var marker_icon = "{{$field['marker_icon']}}";
+                (marker_icon == '') ? null : marker_icon;
+                
+                var map = new google.maps.Map(document.getElementById('map-canvas'), mapOptions);
+            
+                var markers = new google.maps.Marker({
+                    position: latlng,
+                    map: map,
+                    icon : marker_icon
+                });
+            
+                google.maps.event.addListener(map, "click", function (location) {
+                    //map.setCenter(location.latLng); // set location to map's center each time map is clicked
+                    setLatLong(location.latLng.lat(), location.latLng.lng());
+
+                    marker.setMap(null);
+                    markers.setMap(null);
+                
+                    marker = new google.maps.Marker({
+                        position: location.latLng,
+                        map: map,
+                        icon : marker_icon
+                    });
+                    marker.setPosition(location.latLng);
+                    setGeoCoder(location.latLng);
+                });
+
+                var input = (document.getElementById('search_address'));
+
+                var autocomplete = new google.maps.places.Autocomplete(input);
+                autocomplete.bindTo('bounds', map);
+                autocomplete.setTypes([]);
+
+                var infowindow = new google.maps.InfoWindow();
+                marker = new google.maps.Marker({
+                    map: map,
+                    anchorPoint: new google.maps.Point(0, -29),
+                    icon : marker_icon
+                });
+
+                google.maps.event.addListener(autocomplete, 'place_changed', function() {
+                    //infowindow.close();
+                    marker.setVisible(true);
+                    markers.setMap(null);
+                    var place = autocomplete.getPlace();
+
+                    if (!place.geometry) return;
+
+                    // If the place has a geometry, then present it on a map.
+                    if (place.geometry.viewport) {
+                        map.fitBounds(place.geometry.viewport);
+                    } else {
+                        map.setCenter(place.geometry.location);
+                        map.setZoom(default_zoom);
+                    }
+                    marker.setIcon(marker_icon);
+                    map.setZoom(default_zoom);
+                    marker.setPosition(place.geometry.location);
+                    marker.setVisible(true);
+                    setLatLong(place.geometry.location.lat(), place.geometry.location.lng());
+
+                });
+
+                document.getElementById('geolocate').onclick=function(){
+
+                    $('.'+geolocate_icon).addClass('fa-spin'); // add spinning animation for no reason..
+
+                    if(navigator.geolocation) {
```

Comment:
Space after `if`

Reply:
Again, this is all back to the developer's preference

Reply:
https://contribute.jquery.org/style-guide/js/#spacing